hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/dag/scheduler/DagScheduler.java

```diff
@@ -109,12 +109,16 @@ private void executeNode(DagNode node) {
       throw new RuntimeException("DagNode already completed! Cannot re-execute");
     }
     try {
-      log.warn("executing node: " + node.getName() + " of type: " + node.getClass());
-      node.execute(executionContext);
-      node.setCompleted(true);
-      log.info("Finished executing {}", node.getName());
+      int repeatCount = node.getConfig().getRepeatCount();
+      while (repeatCount > 0) {
+        log.warn("executing node: " + node.getName() + " of type: " + node.getClass());
+        node.execute(executionContext);
+        node.setCompleted(true);
```

Comment:
in general repeatCount is confusing. for eg: 
```
first_insert:
  config:
    record_size: 70000
    num_insert_partitions: 1
    repeat_count: 5
    num_records_insert: 1000
  type: InsertNode
  deps: none
```
In this dag yaml, I expect 5*1000 records to be inserted since repeat count is 5. If 5000 records are to be expected, don't we need to fix all yaml dags(complex-dag-mow.yaml etc) for hive query results. I don't see any fixes in this patch for expected record counts though.

Reply:
You're right, I fixed the repeat count and tested it on the unit tests but didn't confirm the DAG under the demo's. I've changed the repeat count to be 1 in the docker DAG so it doesn't cause confusion and have added repeat_count only to the unit_test dags to confirm repeat count works.